apps/distgo/cmd/docker/build_test.go

```diff
@@ -35,6 +35,7 @@ import (
 	"github.com/palantir/godel/apps/distgo/cmd/dist"
 	"github.com/palantir/godel/apps/distgo/cmd/docker"
 	"github.com/palantir/godel/apps/distgo/params"
+	dockerparams "github.com/palantir/godel/apps/distgo/params/docker"
```

Comment:
refactored the code. Now the params package does not have the logic for building the image. moved it to the docker package. This now consistent with dist and build packages.